### PR TITLE
French README localization

### PR DESCRIPTION
**Pull request by le-jun**
Translated the README. Couldn’t find french plaintext versions’ of licences and left them as is. One could copypaste content from the HTML pages, but I’m not sure if that how other did it in their own languages or if there’s a more efficient way.

### CODE REVIEW

File: fr/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+> Pour cela, préférez les licences Free Software telles que GPL, BSD, etc.
+> Les licences Creative Commons s’appliquent uniquement à la propriété intellectuelle.
+
+L’approche la plus simple pour trouver la bonne licence CC pour votre projet est le [site Creative Commons](https://creativecommons.org/choose/).
+
+Nous proposons ici trois de ces licences :
+
+* [Creative Commons Attribution 4.0 International](#cc-attribution-40-international)
+* [Creative Commons Attribution-ShareAlike 4.0 International](#cc-attribution-sharealike-40-international)
+* [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International](#cc-attribution-noncommercial-sharealike-40-international)
```
Comment (santisoler) [inline fr/README.md]: Would you translate the license names as well. There are official translations for each one of them, you can find them in https://creativecommons.org/licenses/?lang=fr

These changes should also be applied on the following sections of the README.md.

File: fr/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+```markdown
+Shield : [![CC BY 4.0][cc-by-shield]][cc-by]
+
+Cette œuvre est mise à disposition selon les termes de la
+[Creative Commons Attribution 4.0 International License][cc-by].
+
+[![CC BY 4.0][cc-by-image]][cc-by]
+
+[cc-by]: http://creativecommons.org/licenses/by/4.0/
```
Comment (santisoler) [inline fr/README.md]: The links to the licenses descriptions should point to their translated versions. For example, this link should point to https://creativecommons.org/licenses/by/4.0/deed.fr

**Review by santisoler — CHANGES_REQUESTED** [review]
Hi @le-jun ! Thanks for the contribution, it's very appreciated!

> Couldn’t find french plaintext versions’ of licences and left them as is. One could copypaste content from the HTML pages, but I’m not sure if that how other did it in their own languages or if there’s a more efficient way.

Sadly there isn't a more efficient way. I've contacted Creative Commons some time ago asking for plain text versions of their licenses in other languages than English. There's an open Issue about it: https://github.com/creativecommons/cc-legal-tools-app/issues/120

While this get solved upstream, if you could create the plain text versions of the licenses in French that would be great. If you don't have the time, we can tackle it.

I'm leaving some comments below.

Thanks again for this PR, a lot of french speakers will surely appreciate it.

File: fr/README.md (hunk 1 above)
Comment (le-jun) [reply fr/README.md]: Done!

File: fr/README.md (hunk 2 above)
Comment (le-jun) [reply fr/README.md]: Done!

**Comment by le-jun** [comment]
> While this get solved upstream, if you could create the plain text versions of the licenses in French that would be great. If you don't have the time, we can tackle it.

That’d be too much for me to do right now, especially considering the softwrap on the official content. Please do take up what’s left of the job. Here are the links to the content, just so you don’t have to put 3 sec of effort to find them:

[cc-by](https://creativecommons.org/licenses/by/4.0/legalcode.fr)
[cc-by-sa](https://creativecommons.org/licenses/by-sa/4.0/legalcode.fr)
[cc-by-nc-sa](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode.fr)

**Review by santisoler — APPROVED** [review]
Thanks @le-jun ! Greatly appreciated!

> That’d be too much for me to do right now, especially considering the softwrap on the official content. Please do take up what’s left of the job.

It's very understandable, no worries. I will tackle it. Thanks for the links!

**Pull request merged**